Cargo.toml

```diff
@@ -7,6 +7,7 @@ members = [
   "src/cmd_all",
   "src/common",
   "src/common/common_service",
+  "src/common/fields-derive",
```

Comment:
prefer underscore, to be consistent with others

Reply:
It’s already inconsistent now, although most of them are underscores, there are still several crates using dash.